edxval/tests/test_api.py

```diff
@@ -2480,6 +2480,37 @@ def test_create_or_update_video_exceptions(self, video_id, file_format, provider
 
         self.assertEqual(transcript_exception.exception.message, exception_message)
 
+    @mock.patch.object(VideoTranscript, 'save')
+    def test_create_or_update_transcript_atomicity(self, mock_save):
```

Comment:
Do we need a test where we don't get the error while saving? (i.e transcript is updated and old file is deleted correctly)